workers/ohsome_quality_analyst/utils/definitions.py

```diff
@@ -1,35 +1,72 @@
-"""
-Global Variables and Functions.
-"""
-
+"""Global Variables and Functions."""
 import glob
 import logging
 import logging.config
 import os
 import sys
+from dataclasses import dataclass
+from types import MappingProxyType
 from typing import Dict, List
 
 import yaml
+from xdg import XDG_DATA_HOME
 
 from ohsome_quality_analyst import __version__ as oqt_version
+from ohsome_quality_analyst.utils.exceptions import RasterDatasetUndefinedError
 from ohsome_quality_analyst.utils.helper import flatten_sequence, get_module_dir
 
 # Dataset names and fid fields which are available in the Geodatabase
-DATASETS = {
-    "regions": {"default": "ogc_fid", "other": ("name",)},
-    "gadm": {
-        "default": "uid",  # ISO 3166-1 alpha-3 country code
-        "other": (
-            *tuple(("name_{0}".format(i) for i in range(6))),
-            *tuple(("id_{0}".format(i) for i in range(6))),
-            *tuple(("gid_{0}".format(i) for i in range(6))),
-        ),
-    },
-}
+DATASETS = MappingProxyType(  # Immutable dict
+    {
+        "regions": {"default": "ogc_fid", "other": ("name",)},
+        "gadm": {
+            "default": "uid",  # ISO 3166-1 alpha-3 country code
+            "other": (
+                *tuple(("name_{0}".format(i) for i in range(6))),
+                *tuple(("id_{0}".format(i) for i in range(6))),
+                *tuple(("gid_{0}".format(i) for i in range(6))),
+            ),
+        },
+    }
+)
 # Dataset names and fid fields which are through the API
 DATASETS_API = DATASETS.copy()
 DATASETS_API.pop("gadm")
 
+
+@dataclass(frozen=True)
+class RasterDataset:
+    """Raster datasets available on disk."""
+
+    name: str
+    filename: str
+    crs: str
+
+
+RASTER_DATASETS = (
```

Comment:
I think some of these datasets would also be available in `4326`. If this would make things easier for OQT, e.g. we need to transform the geometries less often, we could think about using these instead

Reply:
Thats a good idea. At least the GHS_POP seems to be available for 4326. I will make changes.

Reply:
After discussion: Leave it as is (Mollenweid)